### DISCUSSION TITLE
20211112 all small icons disappeared

### DISCUSSION THREAD

**Discussion by chipdipru** [rern/rAudio#608] [Bug Reports]
20211112
all small icons disappeared

![Untitled-1](https://user-images.githubusercontent.com/43340836/141486850-5f276cad-b59a-4e9c-86d8-abb3de9c149c.jpg)

**Comment by klassikmann** [comment]
Can't confirm. Looks good here.

**Comment by tehnostil** [comment]
I confirm that there is a problem. RAudio-1-64bit-20211203.img.xz image. Icons are not displayed on all desktops (browsers chrome 96.0.4664.110, opera 82.0.4227.33, edge 96.0.1054.62). Everything is displayed correctly on the mobile phone.

**Comment by hjfisch** [comment]
- No Problem here on Vivaldi and Opera .
- 
![Screenshot 2021-12-31 080619](https://user-images.githubusercontent.com/93144100/147808994-66241d02-84e9-4db7-b262-e13dba969c7b.jpg)

**Comment by tehnostil** [comment]
![a46fada143](https://user-images.githubusercontent.com/96903863/147851816-e953f98b-7361-467a-a3f7-ebf6d6aabb95.jpg)
chrome on desktop pc

**Comment by tehnostil** [comment]
![9b19967790](https://user-images.githubusercontent.com/96903863/147851854-752c98da-943c-4535-93a9-7ab7d962c99f.jpg)

**Comment by hjfisch** [comment]
Maybe this belongs to your Browser-Settings .
Absolutly no problem with Opera and Vivaldi.
![Screenshot 2022-01-01 145938](https://user-images.githubusercontent.com/93144100/147852315-ff1c37d3-f345-476d-9a3f-ae55ec62dff4.jpg)

**Comment by hjfisch** [comment]
Tried with Microsoft Edge.
Also no Problem.
![Screenshot 2022-01-01 151030](https://user-images.githubusercontent.com/93144100/147852508-49b6bbd8-9e06-4981-a48d-7fd7d45cda84.jpg)

**Comment by tehnostil** [comment]
![d848a46f90](https://user-images.githubusercontent.com/96903863/147852488-24e1577b-9c22-4dc4-819b-ddc6ad40a3a3.jpg)
I installed Vivaldi and the problem persists.

**Comment by hjfisch** [comment]
I've two RPI 4 running with rAudio-1 and tried to reproduce this problem
on 2 different PC's now. But no luck , all running fine .
Did you try another PC , maybe it was caused by the graphic driver .

**Reply by tehnostil** [reply]
I only have one RPI 3b, there is no other

**Comment by momosus** [comment]
This topic is old but I just want to confirm the same problem. I am new to rAudio-1, I tried Firefox, Pale Moon, Waterfox, Chrome and I have the same problem. I tried Firefox on a different computer, same problem.
